### DISCUSSION TITLE
Best method to remove dust from image

### DISCUSSION THREAD

**Discussion by cejones1** [ImageMagick/ImageMagick6#94] [General]
I have some dust on a microscope lens that I cannot remove, but I can take a background image of it as well as an image of my specimen.  The dust shows up as a few darker spots in the specimen image as well as the blank background image I take.  

I tried to simply minus the blank image from the specimen image, but it seems to inverse the image.  

![background](https://user-images.githubusercontent.com/49353144/90572567-fb33f480-e179-11ea-81f8-fec6af51644c.png)
![specimen](https://user-images.githubusercontent.com/49353144/90572568-fb33f480-e179-11ea-9a52-ab8b99bd0c0a.png)

i have tried a number of the mathematically methods to remove the dust but I haven't been able to figure out one that works best.  Has anyone else done something similar or have some advice for me?

Thank you for any help,

**Comment by dlemstra** [comment]
Continued here: https://github.com/ImageMagick/ImageMagick/discussions/2440
